ca/certificate-authority.go

```diff
@@ -215,89 +217,190 @@ func (ca *CertificateAuthorityImpl) RevokeCertificate(serial string, reasonCode
 	ocspResponse, err := ca.OCSPSigner.Sign(signRequest)
 	if err != nil {
 		// AUDIT[ Revocation Requests ] 4e85d791-09c0-4ab3-a837-d3d67e945134
-		ca.log.AuditErr(err)
+		ca.Log.AuditErr(err)
 		return err
 	}
 	err = ca.SA.MarkCertificateRevoked(serial, ocspResponse, reasonCode)
 	return err
 }
 
-// IssueCertificate attempts to convert a CSR into a signed Certificate, while
-// enforcing all policies.
-func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest, regID int64) (core.Certificate, error) {
-	emptyCert := core.Certificate{}
-	var err error
+func uniqueHostNames(csr *x509.CertificateRequest) (commonName string, hostNames []string) {
```

Comment:
Actually, it's not lost, since it's in the policy authority.  But it should also be done here.